cst316/spring16project-Dart
# US-61

Testing Procedure:
1. Create a new project and verify that the new project dialog allows you to enter working hours 0-24 for each day of the week.
2. Go to your memoranda save data directory (on Windows it is C:\Users*YourUsername*.memoranda), open the .projects file with 
   your text editor of choice, and verify that your newly-created project has a workinghrs property with the corresponding values
   you entered for the weekly working hours.
3. Set the new project you created (the one for which you entered working hours) as the current project, then choose File > Pack project… 
   and save the project as a .zip file.  Next, delete the project from Memoranda, then re-import it using File > Unpack project…  Finally, 
   open the .projects file in the .memoranda directory and verify that the project has been properly re-imported with the same values for 
   workinghrs that you had entered previously.

WARNING: This new code assumes that all new projects will have been created with WorkingHours, attempting to pack or unpack old projects 
         created without WorkingHours will cause problems!  If need be you can delete your .memoranda directory and then Memoranda will 
         recreate it the next time it is started, thus ensuring that all projects will be created with WorkingHours.
